assets/cubyz/recipes/special_recipes.zig.zon

```diff
@@ -131,4 +131,12 @@
 		.inputs = .{"cubyz:gravel"},
 		.output = "4 cubyz:pebbles",
 	},
+	.{
+		.inputs = .{"2 cubyz:sulfur_ore", "2 cubyz:coal"},
+		.output = "cubyz:dynamite/stick",
+	},
+	.{
+		.inputs = .{"16 cubyz:dynamite/stick"},
```

Comment:
```suggestion
		.inputs = .{"8 cubyz:dynamite/stick"},
```

With new texture it is not that obvious how much dynamite there is in a block, so I think we can make this cheaper to make.